model/godleyIcon.cc

```diff
@@ -314,19 +314,19 @@ namespace minsky
     float y= this->y() - 0.5*bottomMargin()-0.15*iHeight()*z;
     for (auto& v: m_flowVars)
       {
-        // right justification if displayed, left otherwisw
+        // right justification if displayed, left otherwise. Remove this for ticket 1220?
         v->rotation(0);
-        v->moveTo(x+v->x() - (variableDisplay? v->right(): v->left()), y);
+        v->moveTo(x+v->x() - v->right(), y);
         y+=v->height();
       }
     x= this->x() + 0.55*leftMargin()-0.45*iWidth()*z;
     y= this->y() + 0.5*iHeight()*z-0.5*bottomMargin();
 
     for (auto& v: m_stockVars)
       {
-        // top justification at bottom of icon if displayed, bottom justified otherwise
+        // top justification at bottom of icon if displayed, bottom justified otherwise. Remove this for ticket 1220?
         v->rotation(90);
-        v->moveTo(x, y + v->y() - (variableDisplay? v->top(): v->bottom()));
+        v->moveTo(x, y + v->y() - v->top());
```

Comment:
No - because this will stuff up wires going to the Godley table. See the 1Free example. To solve #1217, we need to disable selecting on subsidiary variables when variableDisplay is false. To solve #1220, we need to disable pulling wires from subsidiary flow variables, which we should probably do regardless of the state of variableDisplay.

Reply:
I thought it was a long shot. Disabling selecting on subsidiary variables would involve more than just VariableBase::clickType, wouldn't it? I think the mouse events in the Canvas class will also be affected by this?